help/en/manual/DecoderPro/Main_VSD.shtml

```diff
@@ -282,6 +281,9 @@
       <p>If checked, the Engine sounds will react immediately to
       Throttle changes, without the Engine Start button first
       having to be pressed.</p>
+      <p><em>Note:</em> Diesel engines normally have a starting-sound. 
+      Throttle changes will not be processed util the starting-sound
```

Comment:
Apologies, but just noticed a typo - 'util' should be 'until'

Reply:
Thanks.